app/views/admin/blog_posts/index.html.erb

```diff
@@ -45,6 +45,8 @@
             </tr>
           <% end %>
         </table>
+        <% @blog_posts = BlogPost.page(params[:page]) %>
```

Comment:
`@blog_posts` is already set in the controller. You don't need to set it again in the view.